e2e/tests/navigation.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { expect, Locator, test as playwrightTest } from '@playwright/test'
+
+import { ALL_PAGES } from './constants'
+
+const test = playwrightTest.extend<{ navigation: Locator }>({
+  navigation: async ({ page }, use) => {
+    await use(page.getByRole('navigation', { name: 'Menu' }))
+  },
+})
+
+test.describe('navigation', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/')
+  })
+  test.describe('@tablet-and-desktop', () => {
```

Comment:
Would be good to keep a newline between tests I think (not sure if dprint can be made to do that automatically, probably not as it's just a function call).